### PR TITLE
add elastic search 6 support

### PR DESCRIPTION
**Pull request by mb3rnard**

### CODE REVIEW

File: src/Resources/envs/symfony/defaults.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ packages:
         enabled:    false
         required:   false
         default:    '5'
```
Comment (chalasr) [inline src/Resources/envs/symfony/defaults.yml]: should we make it the default version?

Comment (mb3rnard) [reply src/Resources/envs/symfony/defaults.yml]: I don't see why we couldn't :) I've made the change

**Comment by chalasr** [comment]
Fixtures needs to be updated :)
`UPDATE_FIXTURES=1 make test` should fix them all

**Review by chalasr — APPROVED** [review]

**Comment by chalasr** [comment]
Thank you @mb3rnard.

**Pull request closed without merging**